aws-es-proxy.go

```diff
@@ -210,6 +212,10 @@ func (p *proxy) getSigner() *v4.Signer {
 }
 
 func (p *proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if p.remoteTerminate && r.URL.Path == "/terminate-proxy" && r.Method == http.MethodPost {
```

Comment:
How did you land on using `POST`?

Reply:
I didn't want to terminate on a GET and while I considered DELETE, this isn't exactly restful, so I figured POST to be the most intuitive.